README.md

```diff
@@ -80,21 +80,24 @@ ______________________________________________________________________
 
 ## Continuous Integration
 
-Lightning is rigorously tested across multiple GPUs, TPUs CPUs and against major Python and PyTorch versions.
+Lightning is rigorously tested across multiple CPUs, GPUs, TPUs, IPUs, and HPUs and against major Python and PyTorch versions.
```

Comment:
We have nearly the same `/src/pytorch_lightning/README.md` as `/README.md`, but is it on purpose? There was a discrepancy between them, so I was wondering if we could remove/adjust either of them.